packages/runtime-core/src/renderer.ts

```
@@ -482,13 +483,13 @@ function baseCreateRenderer(
     }
 
     // set ref
-    if (ref != null && parentComponent) {
+    if (!isNullish(ref) && parentComponent) {
       setRef(ref, n1 && n1.ref, parentSuspense, n2 || n1, !n2)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)